.eslintrc

```diff
@@ -0,0 +1,17 @@
+{
+  "parser": "@typescript-eslint/parser",
```

Comment:
In case you‘re not familiar with @typescript-eslint, it’s now the [official ESLint-supported way of linting TypeScript](https://eslint.org/blog/2019/01/future-typescript-eslint) (ts-lint is deprecated)